templates/Sutil-Safe/src/Client/public/index.html

```diff
@@ -1,13 +0,0 @@
-<!doctype html>
```

Comment:
Looks like the contents of `public` disappeared :smile:

Reply:
corrected - index.html is not in output folder